rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -331,13 +331,22 @@ def test_find_uniswap_v2_lp_token_price(inquirer, globaldb, ethereum_manager):
 @pytest.mark.parametrize('use_clean_caching_directory', [True])
 @pytest.mark.parametrize('should_mock_current_price_queries', [False])
 def test_find_curve_lp_token_price(inquirer_defi, ethereum_manager):
-    address = '0xb19059ebb43466C323583928285a49f558E572Fd'
+    lp_token_addr = '0xA3D87FffcE63B53E0d54fAa1cc983B7eB0b74A9c'
```

Comment:
Why change this address?

Reply:
No particular reason, it was just a bit easier for me to find another pool's info (to put it in cache)